[WIP] v6 Docs Dev (jschalz) (#2025)
loft-sh/devspace: docs/pages/configuration/images/image-definition/dockerfile.mdx

    dockerfile: frontend/Dockerfile
    context: frontend/
```
**Explanation:**

> - The first image would be built using the Dockerfile in `./Dockerfile` and the context path `./`.

Reviewer: ```suggestion
- The first image is built using the Dockerfile in `./Dockerfile` and the context path `./`.
```

> - The second image would be built using the Dockerfile in `./frontend/Dockerfile` and the context path `./frontend/`.

Reviewer: ```suggestion
- The second image is built using the Dockerfile in `./frontend/Dockerfile` and the context path `./frontend/`.
```

> - Switching the `context` for image `frontend` would assure that a statement like `ADD file.txt` or  `COPY file.txt .` in `./frontend/Dockerfile` would use the local file `./frontend/file.txt` instead of `./file.txt`.

Reviewer: ```suggestion
- Switching the `context` for image `frontend` assures that a statement like `ADD file.txt` or  `COPY file.txt .` in `./frontend/Dockerfile` uses the local file `./frontend/file.txt` instead of `./file.txt`.
```

> - In this example, it would probably be useful to have a `./.dockerignore` file which ignores the `frontend/` folder when building the first image called `backend`.

Reviewer: ```suggestion
- In this example, it is useful to have a `./.dockerignore` file ,which ignores the `frontend/` folder when building the first image called `backend`.
```

Author: The comma needs to be moved over from "which" to "file", but otherwise I agree with it.
